src/components/AdminDashboard/AdminDashboard.jsx

```diff
@@ -0,0 +1,90 @@
+
+import './AdminDashboard.module.css';
+
+import { useBackend } from '../../contexts/BackendContext';
+import { useEffect, useState } from 'react';
+import { Box } from '@chakra-ui/react';
+
+
+const BusinessDashboard = () => {
+
+    // Created the use states
+  const { backend } = useBackend();
+  const [donationData, setDonationData] = useState([]);
+  const [businessDictionary, setBusinessDictionary] = useState([]);
+
+
+  useEffect(() => {
+    const getData = async () => {
+      try {
+        // fetches the business table to be used in pending &total #
+        const businessResponse = await backend.get('/business');
+        setBusinessDictionary(businessResponse.data);
+
+        // fetches donation table data
+        const donationResponse = await backend.get('/donation/');
+        setDonationData(donationResponse.data);
+
+      } catch (error) {
+        console.error('Error fetching data:', error);
+      }
+    };
+    getData();
+  }, []);
+
+  // Counts number of donation forms submitted by unique businesses
+  const calculateTotalDonationForms = () => {
+
+    const uniqueBusiness = []
+    for(const [key, value] of Object.entries(donationData)){
+        const businessId = value["business_id"]
+        console.log(key)
+        if (uniqueBusiness.includes(businessId) == false){
+          uniqueBusiness.push(businessId)
+        }
+    }
+
+    return (
+        uniqueBusiness.length
+    );
```

Comment:
you dont need to parenthesize this return statement since its really short, just call `return uniqueBusiness.length` or it would be `uniqueBusiness.size` when you have a set im pretty sure